font/ttf.ksy

```diff
@@ -242,6 +242,112 @@ types:
               - id: repeat_value
                 type: u1
                 if: repeat
+  gvar:
+    doc: >
+      gvar - Glyph Variations Table
+      This table provides all of the variation data that describe how TrueType glyph outlines in a 'glyf' table change across the font’s variation space.
+    seq:
+      - id: major_version
```

Comment:
Use `type: fixed` (I don't know why it is not called `version`) instead of these 2 fields.

Reply:
This layout may be bitwise consistent with 'fixed' but it isn't spec'ed that way. The specification is not 'fixed' (as is the case with the version many other tables) but two uint16. I think this may have even been done on purpose since when this table was defined the 'fixed' versions in other tables were often being serialized incorrectly due to using 'fixed'.